frontend/views/components/avatar-editor/AvatarEditorModal.vue

```diff
@@ -0,0 +1,165 @@
+<template lang="pug">
+modal-template(
+  class='is-centered'
+  ref='modal'
+  data-test='AvatarEditorModal'
+  :a11yTitle='L("Edit avatar")'
+)
+  template(slot='title')
+    i18n Edit avatar
+
+  form.c-form(@submit.prevent='')
+    editor-canvas.c-canvas-container(
+      ref='editorCanvas'
+      :zoom='zoom'
+      @pointer-wheel='HandleWheelOnCanvas'
+      @pinch-in='decrementSlider(3)'
+      @pinch-out='incrementSlider(3)'
+    )
+
+    .c-slider-container
+      button.is-icon-small(@pointerdown='decrementSlider()')
+        i.icon-magnifying-minus
+
+      slider-continuous.c-slider(
+        ref='slider'
+        :hideText='true'
+        uid='avatar-zoom'
+        :min='config.sliderMin'
+        :max='config.sliderMax'
+        :value='form.slider'
+        @input='onSliderInput'
+      )
+
+      button.is-icon-small(@pointerdown='incrementSlider()')
+        i.icon-magnifying-plus
+
+    i18n.has-text-1.hide-touch-device(tag='p') Click and drag to reposition
+    i18n.has-text-1.hide-hoverable-device(tag='p') Pinch to zoom, drag to reposition
+
+    .buttons
+      button.is-outlined(
+        type='button'
+        @click='close'
+      ) Cancel
+
+      button-submit(
+        key='save'
+        data-test='saveBtn'
+        @click='submit'
+      ) Save
```

Comment:
Missing i18n, try with `tag='button'` and `tag='button-submit'`, or include a child `i18n` element